packages/document/main-doc/docs/en/guides/advanced-features/code-split.mdx

```diff
@@ -2,13 +2,12 @@
 title: Code Split
 sidebar_position: 6
 ---
-# Code Split
+# Code Splitting
 
-Code Split is a common way to optimizing front-end resource loading. This doc will introduce three methods supported by Modern.js:
+Code splitting is a common method to optimize frontend resource loading. This article will introduce the three types of code splitting supported by Modern.js:
```

Comment:
Is it better to use "method" than "way"?

Reply:
if there is a specific way, it is better to use method, chatgpt says